skimage/util/compare.py

```diff
@@ -28,9 +28,9 @@ def compare_images(image1, image2, method='diff', *, n_tiles=(8, 8)):
 
     Notes
     -----
-    `diff` computes the absolute difference between the two images.
-    `blend` computes the mean value.
-    `checkerboard` makes tiles of dimension `n_tiles` that display
+    ``'diff'`` computes the absolute difference between the two images.
```

Comment:
We must keep single backquotes now... (I'm responsible for the mess...)

Reply:
@sciunto I think actually in this case this is better, because it's a literal string, not an argument name.

Reply:
You're  right, the rendering will be better.